site/content/docs/0.0/migration.md

```diff
@@ -77,9 +109,13 @@ toc: true
 - <span class="badge text-bg-success">New</span> Shadows utilities:
   `.shadow-none`, `.shadow-raised`, `.shadow-drag`, `.shadow-overlay-default`, `.shadow-overlay-emphasized`, `.shadow-sticky-default`, `.shadow-sticky-emphasized` and `.shadow-sticky-navigation-scrolled`.
 
+#### Sizing
+
+- <span class="badge text-bg-success">New</span> Sizing utility `.mw-none` to set `max-width: none`.
```

Comment:
I don't know if the same mention as the one done in the migration from boosted is useful in here. I'd say yes.